lib/elastomer/version_support.rb

```diff
@@ -37,6 +37,12 @@ def supports_warmers?
       es_version_2_x?
     end
 
+    # COMPATIBILITY: The Tasks API is evolving quickly; features, and request/response\
```

Comment:
looks like there's a extra `\` in this comment

Reply:
thanks!